app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
+  after_save :comment_counter_update
+
   belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  private
+  
+  def comment_counter_update
+    post.increment!(:posts_counter)
```

Comment:
### 🛑 Required change

- [ ] Kindly make this a `:comments_counter` instead of a `:posts_counter`

Reply:
Thanks for your review, suggested changes implemented.